packages/jest-resolve/src/default_resolver.js

```diff
@@ -85,16 +87,6 @@ function resolveSync(x: Path, options: ResolverOptions): Path {
   /*
    * helper functions
    */
```

Comment:
There are two sections of helper functions - those within the `resolveSync()` closure, and the two memoized helpers `isFile` and `isDirectory`.